### PR TITLE
Prepare for 6.5.0

### PR DESCRIPTION
**Pull request by Dridi**
SSIA

### CODE REVIEW

**Review by hermunn — COMMENTED** [review]
In addition to this, the Via header needs an update. See

```
bin/varnishd/cache/cache_req_fsm.c:     http_SetHeader(h, "Via: 1.1 varnish (Varnish/6.4)");
```

**Comment by hermunn** [comment]
Apart from the above, this is ready for merge and release.

File: bin/varnishd/cache/cache_req_fsm.c [hunk 1]
```diff
@@ -141,7 +141,7 @@ Resp_Setup_Deliver(struct req *req)
 	http_PrintfHeader(h, "Age: %.0f",
 	    floor(fmax(0., req->t_prev - oc->t_origin)));
 
-	http_SetHeader(h, "Via: 1.1 varnish (Varnish/6.4)");
+	http_SetHeader(h, "Via: 1.1 varnish (Varnish/6.5)");
```
Comment (Dridi) [inline bin/varnishd/cache/cache_req_fsm.c:144]: Isn't it updated here?

Comment (hermunn) [reply bin/varnishd/cache/cache_req_fsm.c:144]: My mistake, sorry. Merge and we are good.

**Pull request merged**